server/code/setup/cpdb

```diff
@@ -106,7 +108,7 @@ class DbSetup(object):
 
         print(liquibase_options)
 
-        output = run_command("liquibase %s migrate -Dcommunity=%s" % (liquibase_options, self.community))
+        output = run_command("/vagrant/server/bin/liquibase.sh %s migrate -Dcommunity=%s" % (liquibase_options, self.community))
```

Comment:
Since `cpdb` is run from a Satellite installation, the proper thing to do here is to add `liquibase.sh` to the candlepin RPM in a known location. You can reference how `cpdb` and `cpsetup` are added. (Note: there's an extra step added recently to fix the shebangs that won't be necessary for liquibase.sh).

I would suggest a location of `/usr/share/candlepin/liquibase.sh`

Reply:
Done. I used the location you suggested.